devel/folly/Portfile

```diff
@@ -35,13 +36,18 @@ depends_lib-append  port:boost \
                     port:zlib \
                     port:zstd
 
+compiler.blacklist-append {clang < 920} macports-clang-3.* macports-clang-4.0 \
+    macports-clang-5.0
+compiler.fallback-append macports-clang-8.0 macports-clang-7.0 macports-clang-6.0
+
 configure.args-append -DBUILD_SHARED_LIBS=ON
 
 configure.cxxflags-append -std=c++14
 
 # https://github.com/facebook/folly/issues/864
 platform darwin 16 {
     configure.args-append -DCOMPILER_HAS_F_ALIGNED_NEW=OFF
+    compiler.blacklist-append macports-clang-8.0
 }
```

Comment:
Is the above really only needed for darwin 16 only ? Ate you sure it also is not needed on older oses as well, in which case the fix should be applied to all darwin <= 16.

Reply:
I would assume it is needed for darwin <= 16. Any hints on the syntax? `platform {darwin <= 16} {...}`?